test/snippet/range/view/deep_no_param.cpp

```diff
@@ -0,0 +1,24 @@
+#include <vector>
+
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/range/view/deep.hpp>
+#include <seqan3/std/ranges>
+
+namespace seqan3::view
+{
+// You can create a permanent alias:
+inline auto const deep_reverse = deep{std::view::reverse};
+}
+
+int main()
+{
+    using seqan3::operator""_dna5;
+
+    std::vector<seqan3::dna5_vector> foo{"AAATTT"_dna5, "CCCGGG"_dna5};
+
+    auto r = foo | std::view::reverse;                     // == [ [C,C,C,G,G,G], [A,A,A,T,T,T] ]
+
+    // These two are equivalent:
+    auto e = foo | seqan3::view::deep_reverse;             // == [ [T,T,T,A,A,A], [G,G,G,C,C,C] ]
```

Comment:
```suggestion
    auto e = foo | my::deep_reverse;                       // == [ [T,T,T,A,A,A], [G,G,G,C,C,C] ]
```

Users should never open `namespace seqan3` or a sub-namespace. Also in the other examples.